package.json

```diff
@@ -48,7 +48,7 @@
   },
   "dependencies": {
     "@semantic-release/error": "^3.0.0",
-    "@vscode/vsce": "^2.15.0",
+    "@vscode/vsce": "^2.31.1",
```

Comment:
I'll update vsce in a separate PR. You can keep this here while I do that, though.

Reply:
It's done, you can rebase now.